packages/pf-codemods/index.js

```diff
@@ -34,9 +34,6 @@ function printResults(engine, results, format) {
     return false;
   }
 
-  // Don't show warnings
-  results.forEach(result => result.messages = result.messages.filter(message => message.severity === 2));
-
```

Comment:
This setting was making it so that warnings never surfaced before, it has to be removed so that we can issue them.

Reply:
can you think of a reason this was here in the first place?

Reply:
Not off the top of my head, and it was just added in this commit that changed things all over the repo https://github.com/patternfly/pf-codemods/commit/1ab03a0f18667c74ff23812791abb81c05ca6932#diff-06072659dcb0ba81f72d49d18cc77c030ba13d5edacde533742c0dd8ac81730f